airbyte-integrations/connectors/source-wrike/source_wrike/schemas/workflows.json

```diff
@@ -10,10 +10,10 @@
       "type": ["string", "null"]
     },
     "standard": {
-      "type": ["string", "null"]
+      "type": ["string", "boolean"]
```

Comment:
Do all the destinations support multiple data types?

Reply:
typo 👍 
fixed